pkg/kn/core/root.go

```diff
@@ -234,6 +235,12 @@ func initConfigFlags() {
 	var aBool bool
 	aBool = viper.GetBool("lookup-plugins")
 	commands.Cfg.LookupPlugins = &aBool
+
+	// set the Cfg.SinkPrefixes from viper if sink is configured
+	if viper.IsSet("sink") {
+		viper.UnmarshalKey("sink", &commands.Cfg.SinkPrefixes)
```

Comment:
what happens with errors, e.g. when the user misconfigured it ? I think we should propagat an error message which as much context as posisble so that its easy to fix.

Reply:
I added error handling in the new version.